src/openms/source/FORMAT/PepXMLFile.cpp

```diff
@@ -1106,42 +1088,33 @@ namespace OpenMS
           {
             desc += String(aa_mod.massdiff);
           }
-          params_.variable_modifications.push_back(desc);
-        }
-      }
-      else
-      {
-        fixed_modifications_.push_back(aa_mod);
-        if (aa_mod.description != "")
-        {
-          params_.fixed_modifications.push_back(aa_mod.description); // TODO
+          //Modification unknown, but trying to continue as we want to be able to read the rest despite of the modifications but warning this will fail downstream
+          error(LOAD, "Modification '" + String(aa_mod.mass) + "/delta" + String(aa_mod.massdiff) + "' is unknown. Resuming with '" + desc +  "' which will probably fail using the data downstream.");
```

Comment:
In that case it, will represent the mods as they were imported before the fix. e.g. ```C+57.0215``` 
So it prevents a new modification not in modifications db yet to be a complete showstopper for OpenMS.